gum/gumdarwinmodule.c

```diff
@@ -424,8 +424,7 @@ gum_darwin_module_initable_init (GInitable * initable,
     }
   }
 
-  if (self->name == NULL)
-    return gum_darwin_module_ensure_image_loaded (self, error);
+  return gum_darwin_module_ensure_image_loaded (self, error);
```

Comment:
not sure of the impact of this change. would it be possible to just make it lazily  when you need it?

Reply:
By maybe exposing it as a private API you mean?

Reply:
yeah that's an option